tools/cfn2ts/lib/codegen.ts

```diff
@@ -6,6 +6,7 @@ import { upcaseFirst } from './util';
 
 const CORE = genspec.CORE_NAMESPACE;
 const CFN_PARSE = genspec.CFN_PARSE_NAMESPACE;
+const FROM_CFN = genspec.FROM_CFN_NAMESPACE;
```

Comment:
Let's just move `FromCloudFormationOptions` (and `ICfnFinder`) to `cfn-parse.ts` instead, and remove `from-cfn.ts` completely. They were originally in separate files, because `from-cfn.ts` was exported from `core`, as `fromCloudFormation()` was publicly visible. However, with the transition to `fromCloudFormation` being `@internal`, I think there's no reason to separate them anymore.